.atomist/apache2.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
So I was thinking that we might add more license types that we want to use that won't be the same as rug-rugs license